libraries/RainMaker/examples/RMakerCustomAirCooler/ci.json

```diff
@@ -1,6 +1,7 @@
 {
+  "fqbn_append": "PartitionScheme=huge_app",
```

Comment:
For RainMaker I suggest to use "RainMaker 4MB no OTA partition".

Reply:
I just used the same one that was being used by the examples. But I can change that